### PR TITLE
Add why meadowrun page.

### PR DESCRIPTION
**Pull request by kurtschelfthout**

### CODE REVIEW

File: src/html/why.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+@@include("partials/head.html", {"tagline": false})
+
+    <main>
+        <div class="container pt3 pb3">
+
+            <h2>Why Meadowrun?</h2>
+
+            <p>The short answer is: Meadowrun runs your Python code in the cloud, seamlessly. Scale up to a bigger machine, or scale out to many machines – once you are ready to run your analysis or program on a dataset that doesn’t fit on your laptop, Meadowrun helps you do that without hassle and while keeping many of the benefits of running locally.</p>
```
Comment (hrichardlee) [inline src/html/why.html:8]: ~The short answer is~: Meadowrun runs your Python code in the cloud, seamlessly. Scale up to a bigger machine ~,~ or scale out to many machines – once you are ready to run your analysis or program on a dataset that doesn’t fit on your laptop, Meadowrun helps you do that without hassle and while keeping many of the benefits of running locally.

File: src/html/why.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            <br>
+
+            <h3>What’s the Problem?</h3>
+
+            <p>That all sounds great, I hear you say, but don’t we already have containers, AWS/Azure/GCP, Kubernetes and Dask/RAPIDS/Spark? Indeed we do, but putting those together while ensuring an efficient development workflow is no walk in the park.</p>
+
+            <p>For example, containers are very useful for running on scalable compute infrastructure like Kubernetes, but locally you’re more likely to have a virtual environment with third-party packages installed, and you’ll certainly have some local code you’d like to run. How do you get that in the container image efficiently and reproducibly? And if you do get a container running, how do you see the logs? If there are any results how are they sent back?</p>
+
+            <p>Another source of complexity is the wide array of options to choose from, even within a single cloud provider like AWS. Should you allocate an EC2 instance, and if so, should it be a on-demand instance or a spot instance? Can you use Lambdas, or maybe Elastic Container Service? All of these have more options to choose from each with different costs and functional characteristics.</p>
```
Comment (hrichardlee) [inline src/html/why.html:18]: I would maybe just get rid of this last part: ~All of these have more options to choose from each with different costs and functional characteristics.~

I think you're trying to say that even if you decide you're using e.g. ECS, there's still tons more configuration to deal with within that option?

File: src/html/why.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+            <p>Finally, distributed compute clusters like Dask can be a piece of the puzzle, but require setup and management, and again the deployment problem – how do I get my code and dependencies to the cluster workers – is not straightforward to solve.</p>
+
+            <br>
+
+            <h3>How does Meadowrun help?</h3>
+
+            <p>Meadowrun targets Python analytics users who are familiar with notebooks and interactive development, as well as developers who use command line tools like pytest. Example use cases are offloading some compute-intensive analysis or running a long regression test.</p>
```
Comment (hrichardlee) [inline src/html/why.html:26]: Example use cases are offloading ~some~ compute-intensive analyses or ~running~ long regression tests.

File: src/html/why.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+            <p>Another source of complexity is the wide array of options to choose from, even within a single cloud provider like AWS. Should you allocate an EC2 instance, and if so, should it be a on-demand instance or a spot instance? Can you use Lambdas, or maybe Elastic Container Service? All of these have more options to choose from each with different costs and functional characteristics.</p>
+
+            <p>Finally, distributed compute clusters like Dask can be a piece of the puzzle, but require setup and management, and again the deployment problem – how do I get my code and dependencies to the cluster workers – is not straightforward to solve.</p>
+
+            <br>
+
+            <h3>How does Meadowrun help?</h3>
```
Comment (hrichardlee) [inline src/html/why.html:24]: This one and the next one are "Sentence case" but the other two headers above are "Title Case". I think I prefer "Sentence case", but in any case we should be consistent.

File: src/html/why.html [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+            <p>Meadowrun targets Python analytics users who are familiar with notebooks and interactive development, as well as developers who use command line tools like pytest. Example use cases are offloading some compute-intensive analysis or running a long regression test.</p>
+
+            <p>Our guiding principles for Meadowrun are the following.</p>
```
Comment (hrichardlee) [inline src/html/why.html:28]: Our guiding principles for Meadowrun are ~the following.~:

File: src/html/why.html [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+                <li>Meadowrun should need very little setup and ongoing maintenance. To that end, we’ve designed Meadowrun to be serverless – there are no cluster managers, services or databases to set up. Pick a cloud provider, run a simple command and you’re good to go.</li>
+
+                <li>Last but not least – Meadowrun values your time. Interactive development and short feedback loops are important. Meadowrun doesn’t keep you waiting.</li>
+            </ul>
+
+            <br>
+
+            <h3>Where is Meadowrun today?</h3>
+
+            <p>Those are lofty goals and it will take some time to get there, but we hope you’ll follow along for the journey! Today, Meadowrun works with AWS, and can spin up the cheapest cluster of EC2 machines to run a distributed map with your Python function. It automatically and transparently creates an image with your Conda dependencies and local code.</p>
```
Comment (hrichardlee) [inline src/html/why.html:44]: with your Python ~function~ code

**Review by hrichardlee — COMMENTED** [review]
Suggested some line edits, feel free to take whatever you think makes sense

**Comment by kurtschelfthout** [comment]
Re: the quotes, the linter in VSCode says:

The character U+2019 "’" could be confused with the character U+0060 "`", which is more common in source code.

I don't even have 2019 on my keyboard :)

**Comment by hrichardlee** [comment]
I just copy/paste U+2019. I feel like it's pretty common to use this character in HTML, so I kind of disagree with the VS Code linter in this context. But we can always go with `&rsquo;` also.

**Pull request merged**